### DISCUSSION TITLE
essay/2024/03/21/book-event

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#198] [Comments]
# essay/2024/03/21/book-event

작년에 제가 썼던 책을 블로그 독자 여러분들에게 선물해드리려 합니다. 저의 첫 책. 건물주의 기쁨과 슬픔

https://jeho.page/essay/2024/03/21/book-event.html

**Comment by voidFunctionReturn0** [comment]
감사합니다 ㅎㅎ!

**Comment by ziw8** [comment]
결심해주셔서 감사합니다.

**Comment by rainingdaewoo** [comment]
무슨 벌써 마감...ㅠㅠ
